src/mug.gleam

```diff
@@ -3,18 +3,87 @@ import gleam/dynamic.{type Dynamic}
 import gleam/erlang/atom
 import gleam/erlang/charlist.{type Charlist}
 import gleam/erlang/process
+import gleam/list
+import gleam/option.{type Option, None, Some}
+import gleam/result
+import gleam/string
+import mug/internal/ssl_options.{
+  type SslOptionName, Cacertfile, Cacerts, CertsKeys, Verify,
```

Comment:
Still have a mix of SSL and TLS terminology here. Which is more correct? I think it is TLS?